include/deal.II/lac/tensor_product_matrix.h

```diff
@@ -281,6 +282,17 @@ namespace internal
     struct MatrixPairComparator
     {
       using MatrixPairType = std::pair<Table<2, Number>, Table<2, Number>>;
+      using VectorizedArrayTrait =
+        dealii::internal::VectorizedArrayTrait<Number>;
+      using ScalarNumber = typename VectorizedArrayTrait::value_type;
+      static constexpr std::size_t width = VectorizedArrayTrait::width;
+
+      MatrixPairComparator()
+        : eps(std::pow<ScalarNumber>(
+            static_cast<ScalarNumber>(10.0),
+            static_cast<ScalarNumber>(
+              std::log10(std::numeric_limits<ScalarNumber>::epsilon()) / 2.0)))
```

Comment:
I know this is pre-existing, but isn't this the same as `std::sqrt(std::numeric_limits<ScalarNumber>::epsilon())`?